Cookbook/Technical-Documents/TechnicalDebt.md

```diff
@@ -7,62 +7,148 @@ Technical debt and legacy code has a sad tendency to build up in any software pr
+| AppointmentReplayViewModel | CNSMR-962, CNSMR-963 | | |
+| AppointmentListViewModel | CNSMR-964, CNSMR-965 | | Used when `isNewAppointmentsEnabled == false`|
+| BookAppointmentViewModel | CNSMR-966, CNSMR-967 | | |
+| PractitionerDetailsViewModel | CNSMR-968, CNSMR-969 | | |
+| ChatHistoryViewModel | CNSMR-970, CNSMR-971 | | |
+| MedicalHistoryViewModel | CNSMR-972, CNSMR-973 | | |
+| PersonalDetailsViewModel | CNSMR-974, CNSMR-975 | | |
+| InfoItemsViewModel | CNSMR-976, CNSMR-977 | | |
+| GPDetailsViewModel | CNSMR-978, CNSMR-979 | | |
+| AddAddressViewModel | CNSMR-980, CNSMR-981 | | |
+| PrivacySettingsViewModel | AV-334 | | |
+| ChooseCountryViewModel | AV-338 | | |
+| ForgotPasswordViewModel | CNSMR-904 | | Used when `isNewForgotPasswordEnabled == false` |
+| NotificationsViewModel | AV-342 | | Used when `isNewNotificationsEnabled == false` |
+| SignInViewModel | AV-332 | | Used when `isNewSigninEnabled == false`|
+| NHSRegistrationStep1ViewController | NRX-361 | | Used when `isV4NHSRegistrationStep1And2Enabled == false` |
+| NHSRegistrationStep2ViewController | NRX-186 | | Used when `isV4NHSRegistrationStep1And2Enabled == false` |
+| RedemptionViewModel | CNSMR-921 | | Used when `isNewRedemptionEnabled ==false ` |
+| OnboardingViewModel | AV-118 | | Used when `isNewWelcomeScreenEnabled == false` |
+
+## Bento View Controllers that need to be Updated to use the Design Library.
+
+`Bento` and its component library `BentoKit` predates the design library. Consequenly there are a number of `BoxRenderer` classes that need to be refactored to employ the design library.
+
+### Renderers that do not employ the design library
+
+| Renderer | JIRA tickets | Code Ownership | Comments |
+| ---------- | -------- | -------- |  -------- |
+| PrescriptionRenderer | | | |
+| PrescriptionDeliveryOptionsRenderer | | | |
+| ChatMenuRenderer | | | |
+| SymptomSelectorRenderer | | | |
+| MessageReportRenderer | | | |
+| PGMConditionDetailsRenderer | | | |
+| PGMConditionDetailsRenderer | | | |
+| ClinicalRecordsRenderer | | | |
+| PGMReportRenderer | | | |
+| GPAtHandDetailsRenderer | | | |
+| PharmaciesRenderer | | | |
+| ClinicalRecordsRenderer | | | |
+| MetricDetailsRenderer | | | |
+| QuestionnaireRenderer | | | |
+| UserFeedbackRatingRenderer | | | |
+| MapSearchResultsRenderer | | | |
+| PlaceListRenderer | | | |
+| ChooseAssessmentRenderer | | | |
+| DetailRenderer | | | |
+| HealthOverviewRenderer | | | |
+| RiskSortMenuRenderer | | | |
+| DiseaseRisksRenderer | | | |
+| ActionsRenderer | | | |
+| AdditionalInfoFormRenderer | | | |
+| RatingRenderer | | | |
+| ChooseStateRenderer | | | |
+| AddressListRenderer | | | |
+| ShopConfirmationRenderer | | | |
+| ChooseProviderRenderer | | | |
+| PaymentPlansRenderer | | | |
+| SubscriptionConfirmationRenderer | | | |
+| SubscriptionsRenderer | | | |
+| EligibilityCheckRenderer | | | |
+| NHSSwitchV2Renderer | | | |
+| NHSConfirmationSwitchRenderer | | | |
```

Comment:
This one is the legacy version of NHSSwitchV2Renderer. I'm asking if we can delete it.

Reply:
It's gone. Could you please remove it?

Reply:
Great news. I will remove it from the list as soon as you have deleted it from the code base 😸.

Reply:
Remove `NHSConfirmationSwitchRenderer`.